### PR TITLE
Feature/better logging

### PR DESCRIPTION
**Pull request by jechols**
Logging to a file now works properly.  Logging configs no longer clobber each other.

## Submitter Checklist

- [x] Verify all tests pass
  - Run tests with `docker-compose -f test-compose.yml -p onitest up test`
- [x] Update `README.md` and `docs/` as necessary
  - [x] If a `manage.py` command is added, deleted, or modified its help text must
    be valid and it should be documented in detail in
    `docs/advanced/admin-commands.md`
- Update `CHANGELOG.md`
  - [x] Describe change(s) in appropriate section(s)
  - [x] List self in Contributors section
- If a release PR:
  - [x] In `CHANGELOG.md`, replace `[Unreleased]` with version and update compare link
  - [x] Update `core/version.py` with new version
- [x] Resolve merge conflicts
- [x] @mention individual(s) you would like to review the PR
  - Reviews for releases must come from a reviewer at another institution

## Reviewer Checklist

- [x] Verify all tests pass
- [x] Review changes for behavior, bugs, and compliance with [Development
  Standards](https://github.com/open-oni/open-oni/tree/dev/CONTRIBUTING.md#development-standards)
  - Request the submitter make any changes rather than pushing changes yourself.
    Otherwise ask another reviewer to look at any changes you have made.
- [x] If a release, follow the [Release
  Checklist](https://github.com/open-oni/open-oni/tree/dev/CONTRIBUTING.md#release-checklist)

### CODE REVIEW

File: onisite/settings_local_example.py [hunk 1]
```diff
@@ -176,31 +209,6 @@
...
-                }
-            },
-            'handlers': {
-                'console': {
-                    'level': 'DEBUG',
-                    'filters': ['require_debug_true'],
-                    'class': 'logging.StreamHandler',
-                }
-            },
-            'loggers': {
```
Comment (jduss4) [inline onisite/settings_local_example.py:195]: out of curiosity, why did you remove this in the new version?

**Comment by jduss4** [comment]
YESSSS THIS IS WORKING!! I am very excited by the idea that now we will be able to see actual error messages.  I also tested adding a new logger via `import logging` and it's printing to the output!  Yay!!!

File: onisite/settings_local_example.py (hunk 1 above)
Comment (jechols) [reply onisite/settings_local_example.py:195]: It's actually basically been moved.  I combined all registered handlers and filters into the main `LOGGING` definition, so that we didn't have to futz around with large blocks inside the `if` blocks.  The prior approach was a bit unruly to read, but also was really annoying because it was a "clobber the old definition" system.  e.g., if you wanted SQL to be logged, you couldn't have the file-based logging at the same time.

Comment (jduss4) [reply onisite/settings_local_example.py:195]: Oh I see, I had noticed `'loggers': {},` first, but I see you're adding it back in later with a different handler.  NM

**Comment by jechols** [comment]
> Tested this out and it works great, but I found one thing missing. `ONI_LOG_LEVEL` also needs to be added to the list of env vars passed through Docker Compose in `docker-compose.yml` +1

Urgh... yeah, missed that.  I must have put it into my compose override file when testing.
